composer.json

```diff
@@ -16,16 +16,18 @@
         }
     ],
     "require": {
-        "php": ">=5.3.2",
-        "symfony/finder": ">=2.0",
-        "symfony/yaml": ">=2.0",
-        "kriswallsmith/assetic": ">= 1.0",
-        "symfony/framework-bundle": ">=2.0.9"
+        "php": "^7.1",
+        "symfony/finder": "^3.4",
+        "symfony/yaml": "^3.4",
+        "kriswallsmith/assetic": "^1.4",
+        "symfony/translation": "^3.4",
+        "symfony/framework-bundle": "^3.4",
+        "symfony/templating": "^3.4"
```

Comment:
`symfony/templating` and `symfony/translation` are no longer implicitly required by `symfony/framework-bundle`.

Reply:
Should probably lower the PHP version again to keep support for 5.x, so no upgrade is forced.